package.json

```diff
@@ -43,6 +43,9 @@
     "is-typedarray": "~1.0.0",
     "har-validator": "~2.0.2"
   },
+  "optionalDependencies": {
```

Comment:
Can't do that as npm2 installs the optional dependencies by default. Specifying the dependency in the docs should be enough.